packages/grafana-ui/src/components/Combobox/Combobox.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Combobox, Option } from './Combobox';
```

Comment:
It seems like eslint is complaining too.
```suggestion
import userEvent from '@testing-library/user-event';

import { Combobox, Option } from './Combobox';
```
This can also be fixed by running `npx eslint --fix packages/grafana-ui/src/components/Combobox/Combobox.test.tsx`